action.yml

```diff
@@ -14,10 +14,13 @@ inputs:
   version:
     description: 'version to install, default is latest'
     required: false
-  omnitruckUrl:
```

Comment:
Can we keep this and if this is set to something different then default we use `omnitruckUrl` otherwise we use `chefDownloadUrl` to maintain backwards compatability, else when this is merged I fear a few customers will have things break